kiwi/schema/kiwi.rnc

```diff
@@ -1254,6 +1254,16 @@ div {
         ## Specify whether or not this repository should be configured in the
         ## resulting image. Boolean value true or false, the default is false.
         attribute imageinclude { xsd:boolean }
+    k.repository.repository_gpgcheck.attribute =
+        ## Specify whether or not this specific repository is configured to
+        ## to run repository signature validation. If not set, no value is
+        ## appended into the repository configuration file.
+        attribute repository_gpgcheck { xsd:boolean }
+    k.repository.package_gpgcheck.attribute =
+        ## Specify whether or not this specific repository is configured to
+        ## to run package signature validation. If not set, no value is
+        ## appended into the repository configuration file.
+        attribute package_gpgcheck { xsd:boolean }
```

Comment:
One addition to the schema docs. We should fix the following:

```
#==========================================
# common element <rpm-check-signatures>
#
div {
    k.rpm-check-signatures.content = xsd:boolean
    k.rpm-check-signatures.attlist = empty
    k.rpm-check-signatures =
        ## Setup a Package Signature
        [
        db:para [
            "Setup if the package manager should check the package\x{a}"~
            "signature or not. This option could be ignored according\x{a}"~
            "to the used package manager."
        ]
        ]
        element rpm-check-signatures {...
```

* delete the db:para, we don't use that anymore since the docs are not docbook related anymore
* update the very short comment, explaining that this is the system wide package manager setup and can be overwritten by repository specific settings. Also I assume repo specific settings are higher in priority and overwrites the system wide setup iirc